rename openvpn -> wg-openvpnIndex: openvpn-2.3.2/debian/dirs
===================================================================
Index: openvpn-2.3.2/debian/dirs
===================================================================
--- openvpn-2.3.2.orig/debian/dirs	2014-02-04 23:29:42.000000000 +0900
+++ openvpn-2.3.2/debian/dirs	2016-04-14 13:22:16.832171685 +0900
@@ -1,10 +1,10 @@
-etc/openvpn
+etc/wg-openvpn
 etc/network/if-up.d
 etc/network/if-down.d
 etc/bash_completion.d
 usr/sbin
 usr/share/man/man8
-usr/share/doc/openvpn
-usr/share/openvpn
-usr/lib/openvpn
-usr/include/openvpn
+usr/share/doc/wg-openvpn
+usr/share/wg-openvpn
+usr/lib/wg-openvpn
+usr/include/wg-openvpn
Index: openvpn-2.3.2/debian/openvpn.bash_completion
===================================================================
--- openvpn-2.3.2.orig/debian/openvpn.bash_completion	2014-02-04 23:29:42.000000000 +0900
+++ openvpn-2.3.2/debian/openvpn.bash_completion	2016-04-14 13:34:39.496150091 +0900
@@ -1,7 +1,7 @@
-# bash completion for openvpn init.d script
+# bash completion for wg-openvpn init.d script
 # Written by Alberto Gonzalez Iniesta <agi@inittab.org>
 
-_openvpn()
+_wg_openvpn()
 {
 
   local cur 
@@ -10,14 +10,14 @@
   cur=${COMP_WORDS[COMP_CWORD]}
 
   if [ $COMP_CWORD -eq 1 ] ; then
-    COMPREPLY=( $( compgen -W '$( /etc/init.d/openvpn 2>&1 \
+    COMPREPLY=( $( compgen -W '$( /etc/init.d/wg-openvpn 2>&1 \
               | cut -d"{" -f2 | tr -d "}" | tr "|" " " )' -- $cur ) )
   else
-    COMPREPLY=( $( compgen -W '$( command ls /etc/openvpn/*.conf 2>/dev/null \
-              | sed -e 's%/etc/openvpn/%%' -e 's/\.conf//' )' -- $cur ) )
+    COMPREPLY=( $( compgen -W '$( command ls /etc/wg-openvpn/*.conf 2>/dev/null \
+              | sed -e 's%/etc/wg-openvpn/%%' -e 's/\.conf//' )' -- $cur ) )
   fi
 
 }
 
 
-complete -F _openvpn /etc/init.d/openvpn
+complete -F _wg_openvpn /etc/init.d/wg-openvpn
Index: openvpn-2.3.2/debian/openvpn.if-down.d
===================================================================
--- openvpn-2.3.2.orig/debian/openvpn.if-down.d	2014-02-04 23:29:42.000000000 +0900
+++ openvpn-2.3.2/debian/openvpn.if-down.d	2016-04-14 13:35:25.968148740 +0900
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-OPENVPN=/etc/init.d/openvpn
+OPENVPN=/etc/init.d/wg-openvpn
 
 if [ ! -x $OPENVPN ]; then
   exit 0
Index: openvpn-2.3.2/debian/postinst
===================================================================
--- openvpn-2.3.2.orig/debian/postinst	2014-02-04 23:29:42.000000000 +0900
+++ openvpn-2.3.2/debian/postinst	2016-04-14 13:41:39.080137891 +0900
@@ -11,33 +11,33 @@
 
 case "$1" in
   configure)
-    db_get openvpn/create_tun || RET="false"
+    db_get wg-openvpn/create_tun || RET="false"
     if [ "$RET" = "true" ]; then
       [ -e /dev/net ] || mkdir -m 755 /dev/net
       [ -d /dev/net -a ! -e /dev/net/tun ] && mknod /dev/net/tun c 10 200
     fi
-    # move /run/openvpn.foo.pid to (subdir) /run/openvpn/foo.pid
+    # move /run/wg-openvpn.foo.pid to (subdir) /run/wg-openvpn/foo.pid
     if [ -n "$2" ] && dpkg --compare-versions "$2" lt 2.3.2-6 ; then
-      mkdir -p /run/openvpn
-      for f in $(ls /var/run/openvpn.*.pid /var/run/openvpn.*.status 2>/dev/null); do
-        mv "$f" "/run/openvpn/$(basename $f | sed 's/^openvpn\.//')"
+      mkdir -p /run/wg-openvpn
+      for f in $(ls /var/run/wg-openvpn.*.pid /var/run/wg-openvpn.*.status 2>/dev/null); do
+        mv "$f" "/run/wg-openvpn/$(basename $f | sed 's/^wg-openvpn\.//')"
       done
     fi
-    for f in /run/sendsigs.omit.d/openvpn.*.pid; do
+    for f in /run/sendsigs.omit.d/wg-openvpn.*.pid; do
       [ -e "$f" ] && continue
-      b="$(basename $f|sed 's/openvpn\.//')"
+      b="$(basename $f|sed 's/wg-openvpn\.//')"
       # make sure is a vpn name, not *.pid (#730679)
       echo "$b" | grep -q '\*' && continue
-      ln -sf "/run/openvpn/$b" "/run/sendsigs.omit.d/openvpn.$b"
+      ln -sf "/run/wg-openvpn/$b" "/run/sendsigs.omit.d/wg-openvpn.$b"
     done
   ;;
 esac
 
-if [ -x "/etc/init.d/openvpn" ]; then
+if [ -x "/etc/init.d/wg-openvpn" ]; then
 	if [ -x /usr/sbin/invoke-rc.d ]; then
-	   invoke-rc.d openvpn cond-restart || invoke-rc.d openvpn restart
+	   invoke-rc.d wg-openvpn cond-restart || invoke-rc.d wg-openvpn restart
 	else
-	   /etc/init.d/openvpn cond-restart || /etc/init.d/openvpn restart
+	   /etc/init.d/wg-openvpn cond-restart || /etc/init.d/wg-openvpn restart
 	fi
 fi
 
Index: openvpn-2.3.2/debian/update-resolv-conf
===================================================================
--- openvpn-2.3.2.orig/debian/update-resolv-conf	2016-04-14 13:43:26.352134772 +0900
+++ openvpn-2.3.2/debian/update-resolv-conf	2016-04-14 13:43:51.028134054 +0900
@@ -1,14 +1,14 @@
 #!/bin/bash
 # 
-# Parses DHCP options from openvpn to update resolv.conf
-# To use set as 'up' and 'down' script in your openvpn *.conf:
-# up /etc/openvpn/update-resolv-conf
-# down /etc/openvpn/update-resolv-conf
+# Parses DHCP options from wg-openvpn to update resolv.conf
+# To use set as 'up' and 'down' script in your wg-openvpn *.conf:
+# up /etc/wg-openvpn/update-resolv-conf
+# down /etc/wg-openvpn/update-resolv-conf
 #
 # Used snippets of resolvconf script by Thomas Hood and Chris Hanson.
 # Licensed under the GNU GPL.  See /usr/share/common-licenses/GPL. 
 # 
-# Example envs set from openvpn:
+# Example envs set from wg-openvpn:
 #
 #     foreign_option_1='dhcp-option DNS 193.43.27.132'
 #     foreign_option_2='dhcp-option DNS 193.43.27.133'
@@ -49,10 +49,10 @@
         	R="${R}nameserver $NS
 "
 	done
-	echo -n "$R" | /sbin/resolvconf -a "${dev}.openvpn"
+	echo -n "$R" | /sbin/resolvconf -a "${dev}.wg-openvpn"
 	;;
   down)
-	/sbin/resolvconf -d "${dev}.openvpn"
+	/sbin/resolvconf -d "${dev}.wg-openvpn"
 	;;
 esac
 
Index: openvpn-2.3.2/debian/prerm
===================================================================
--- openvpn-2.3.2.orig/debian/prerm	2014-02-04 23:29:42.000000000 +0900
+++ openvpn-2.3.2/debian/prerm	2016-04-14 13:44:16.332133318 +0900
@@ -8,11 +8,11 @@
 test $DEBIAN_SCRIPT_DEBUG && set -v -x
 
 stop_vpn () {
-  if [ -x "/etc/init.d/openvpn" ]; then
+  if [ -x "/etc/init.d/wg-openvpn" ]; then
           if [ -x /usr/sbin/invoke-rc.d ] ; then
-                  invoke-rc.d openvpn stop
+                  invoke-rc.d wg-openvpn stop
           else
-                  /etc/init.d/openvpn stop
+                  /etc/init.d/wg-openvpn stop
           fi
   fi
 }
Index: openvpn-2.3.2/debian/config
===================================================================
--- openvpn-2.3.2.orig/debian/config	2014-02-04 23:29:42.000000000 +0900
+++ openvpn-2.3.2/debian/config	2016-04-14 13:44:41.324132592 +0900
@@ -11,7 +11,7 @@
 
 # Do we want to create /dev/net/tun?
 if [ ! -e /run/udev ] && [ ! -e /dev/net/tun ]; then
-  db_input medium openvpn/create_tun || true
+  db_input medium wg-openvpn/create_tun || true
   db_go
 fi
 
Index: openvpn-2.3.2/debian/default
===================================================================
--- openvpn-2.3.2.orig/debian/default	2014-02-04 23:29:42.000000000 +0900
+++ openvpn-2.3.2/debian/default	2016-04-14 13:45:09.444131774 +0900
@@ -1,28 +1,28 @@
-# This is the configuration file for /etc/init.d/openvpn
+# This is the configuration file for /etc/init.d/wg-openvpn
 
 #
 # Start only these VPNs automatically via init script.
 # Allowed values are "all", "none" or space separated list of
 # names of the VPNs. If empty, "all" is assumed.
 # The VPN name refers to the VPN configutation file name.
-# i.e. "home" would be /etc/openvpn/home.conf
+# i.e. "home" would be /etc/wg-openvpn/home.conf
 #
 #AUTOSTART="all"
 #AUTOSTART="none"
 #AUTOSTART="home office"
 #
 # Refresh interval (in seconds) of default status files
-# located in /var/run/openvpn.$NAME.status
+# located in /var/run/wg-openvpn.$NAME.status
 # Defaults to 10, 0 disables status file generation
 #
 #STATUSREFRESH=10
 #STATUSREFRESH=0
-# Optional arguments to openvpn's command line
+# Optional arguments to wg-openvpn's command line
 OPTARGS=""
 #
-# If you need openvpn running after sendsigs, i.e.
+# If you need wg-openvpn running after sendsigs, i.e.
 # to let umountnfs work over the vpn, set OMIT_SENDSIGS
-# to 1 and include umountnfs as Required-Stop: in openvpn's
+# to 1 and include umountnfs as Required-Stop: in wg-openvpn's
 # init.d script (remember to run insserv after that)
 #
 OMIT_SENDSIGS=0
Index: openvpn-2.3.2/debian/templates
===================================================================
--- openvpn-2.3.2.orig/debian/templates	2014-02-04 23:29:42.000000000 +0900
+++ openvpn-2.3.2/debian/templates	2016-04-14 13:45:46.708130691 +0900
@@ -7,11 +7,11 @@
 # Even minor modifications require translation updates and such
 # changes should be coordinated with translators and reviewers.
 
-Template: openvpn/create_tun
+Template: wg-openvpn/create_tun
 Type: boolean
 Default: false
 _Description: Create the TUN/TAP device?
  If you choose this option, the /dev/net/tun device
- needed by OpenVPN will be created.
+ needed by Wg-Openvpn will be created.
  .
  You should not choose this option if you're using devfs.
Index: openvpn-2.3.2/debian/openvpn.init.d
===================================================================
--- openvpn-2.3.2.orig/debian/openvpn.init.d	2014-02-04 23:30:40.000000000 +0900
+++ openvpn-2.3.2/debian/openvpn.init.d	2016-04-14 13:47:25.324127823 +0900
@@ -1,7 +1,7 @@
 #!/bin/sh -e
 
 ### BEGIN INIT INFO
-# Provides:          openvpn
+# Provides:          wg-openvpn
 # Required-Start:    $network $remote_fs $syslog
 # Required-Stop:     $network $remote_fs $syslog
 # Should-Start:      network-manager
@@ -10,23 +10,23 @@
 # X-Interactive:     true
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
-# Short-Description: Openvpn VPN service
-# Description: This script will start OpenVPN tunnels as specified
-#              in /etc/default/openvpn and /etc/openvpn/*.conf
+# Short-Description: Wg-Openvpn VPN service
+# Description: This script will start Wg-Openvpn tunnels as specified
+#              in /etc/default/wg-openvpn and /etc/wg-openvpn/*.conf
 ### END INIT INFO
 
 # Original version by Robert Leslie
 # <rob@mars.org>, edited by iwj and cs
-# Modified for openvpn by Alberto Gonzalez Iniesta <agi@inittab.org>
+# Modified for wg-openvpn by Alberto Gonzalez Iniesta <agi@inittab.org>
 # Modified for restarting / starting / stopping single tunnels by Richard Mueller <mueller@teamix.net>
 
 . /lib/lsb/init-functions
 
 test $DEBIAN_SCRIPT_DEBUG && set -v -x
 
-DAEMON=/usr/sbin/openvpn
+DAEMON=/usr/sbin/wg-openvpn
 DESC="virtual private network daemon(s)"
-CONFIG_DIR=/etc/openvpn
+CONFIG_DIR=/etc/wg-openvpn
 test -x $DAEMON || exit 0
 test -d $CONFIG_DIR || exit 0
 
@@ -34,8 +34,8 @@
 AUTOSTART="all"
 STATUSREFRESH=10
 OMIT_SENDSIGS=0
-if test -e /etc/default/openvpn ; then
-  . /etc/default/openvpn
+if test -e /etc/default/wg-openvpn ; then
+  . /etc/default/wg-openvpn
 fi
 
 start_vpn () {
@@ -51,21 +51,21 @@
       # status file already given in config file
       STATUSARG=""
     elif test $STATUSREFRESH -eq 0 ; then
-      # default status file disabled in /etc/default/openvpn
+      # default status file disabled in /etc/default/wg-openvpn
       STATUSARG=""
     else
       # prepare default status file
-      STATUSARG="--status /run/openvpn/$NAME.status $STATUSREFRESH"
+      STATUSARG="--status /run/wg-openvpn/$NAME.status $STATUSREFRESH"
     fi
 
-    mkdir -p /run/openvpn
+    mkdir -p /run/wg-openvpn
 
     # tun using the "subnet" topology confuses the routing code that wrongly
     # emits ICMP redirects for client to client communications
     SAVED_DEFAULT_SEND_REDIRECTS=0
     if grep -q '^[[:space:]]*dev[[:space:]]*tun' $CONFIG_DIR/$NAME.conf && \
        grep -q '^[[:space:]]*topology[[:space:]]*subnet' $CONFIG_DIR/$NAME.conf ; then
-        # When using "client-to-client", OpenVPN routes the traffic itself without
+        # When using "client-to-client", Wg-Openvpn routes the traffic itself without
         # involving the TUN/TAP interface so no ICMP redirects are sent
         if ! grep -q '^[[:space:]]*client-to-client' $CONFIG_DIR/$NAME.conf ; then
             sysctl -w net.ipv4.conf.all.send_redirects=0 > /dev/null
@@ -88,12 +88,12 @@
 
     STATUS=0
     start-stop-daemon --start --quiet --oknodo \
-        --pidfile /run/openvpn/$NAME.pid \
-        --exec $DAEMON -- $OPTARGS --writepid /run/openvpn/$NAME.pid \
+        --pidfile /run/wg-openvpn/$NAME.pid \
+        --exec $DAEMON -- $OPTARGS --writepid /run/wg-openvpn/$NAME.pid \
         $DAEMONARG $STATUSARG --cd $CONFIG_DIR \
         --config $CONFIG_DIR/$NAME.conf $script_security < /dev/null || STATUS=1
 
-    [ "$OMIT_SENDSIGS" -ne 1 ] || ln -s /run/openvpn/$NAME.pid /run/sendsigs.omit.d/openvpn.$NAME.pid
+    [ "$OMIT_SENDSIGS" -ne 1 ] || ln -s /run/wg-openvpn/$NAME.pid /run/sendsigs.omit.d/wg-openvpn.$NAME.pid
 
     # Set the back the original default value of send_redirects if it was changed
     if [ "$SAVED_DEFAULT_SEND_REDIRECTS" -ne 0 ]; then
@@ -105,8 +105,8 @@
       --pidfile $PIDFILE --exec $DAEMON --retry 5
   if [ "$?" -eq 0 ]; then
     rm -f $PIDFILE
-    [ "$OMIT_SENDSIGS" -ne 1 ] || rm -f /run/sendsigs.omit.d/openvpn.$NAME.pid
-    rm -f /run/openvpn/$NAME.status 2> /dev/null
+    [ "$OMIT_SENDSIGS" -ne 1 ] || rm -f /run/sendsigs.omit.d/wg-openvpn.$NAME.pid
+    rm -f /run/wg-openvpn/$NAME.status 2> /dev/null
     log_end_msg 0
   else
     log_failure_msg "  Unable to stop VPN '$NAME'"
@@ -162,7 +162,7 @@
 stop)
   log_action_begin_msg "Stopping $DESC"
   if test -z "$2" ; then
-    for PIDFILE in `ls /run/openvpn/*.pid 2> /dev/null`; do
+    for PIDFILE in `ls /run/wg-openvpn/*.pid 2> /dev/null`; do
       NAME=`echo $PIDFILE | cut -c14-`
       NAME=${NAME%%.pid}
       log_daemon_msg "  Stopping VPN '$NAME'"
@@ -174,9 +174,9 @@
   else
     while shift ; do
       [ -z "$1" ] && break
-      if test -e /run/openvpn/$1.pid ; then
+      if test -e /run/wg-openvpn/$1.pid ; then
         log_daemon_msg "  Stopping VPN '$1'"
-        PIDFILE=`ls /run/openvpn/$1.pid 2> /dev/null`
+        PIDFILE=`ls /run/wg-openvpn/$1.pid 2> /dev/null`
         NAME=`echo $PIDFILE | cut -c14-`
         NAME=${NAME%%.pid}
         stop_vpn
@@ -189,10 +189,10 @@
 # Only 'reload' running VPNs. New ones will only start with 'start' or 'restart'.
 reload|force-reload)
   log_action_begin_msg "Reloading $DESC"
-  for PIDFILE in `ls /run/openvpn/*.pid 2> /dev/null`; do
+  for PIDFILE in `ls /run/wg-openvpn/*.pid 2> /dev/null`; do
     NAME=`echo $PIDFILE | cut -c14-`
     NAME=${NAME%%.pid}
-# If openvpn if running under a different user than root we'll need to restart
+# If wg-openvpn if running under a different user than root we'll need to restart
     if egrep '^[[:blank:]]*user[[:blank:]]' $CONFIG_DIR/$NAME.conf > /dev/null 2>&1 ; then
       log_daemon_msg "  Stopping VPN '$NAME'"
       stop_vpn
@@ -211,7 +211,7 @@
 # Only 'soft-restart' running VPNs. New ones will only start with 'start' or 'restart'.
 soft-restart)
   log_action_begin_msg "Soft-restarting $DESC"
-  for PIDFILE in `ls /run/openvpn/*.pid 2> /dev/null`; do
+  for PIDFILE in `ls /run/wg-openvpn/*.pid 2> /dev/null`; do
     NAME=`echo $PIDFILE | cut -c14-`
     NAME=${NAME%%.pid}
     log_daemon_msg "  Soft-restarting VPN '$NAME'"
@@ -229,7 +229,7 @@
   ;;
 cond-restart)
   log_action_begin_msg "Restarting $DESC"
-  for PIDFILE in `ls /run/openvpn/*.pid 2> /dev/null`; do
+  for PIDFILE in `ls /run/wg-openvpn/*.pid 2> /dev/null`; do
     NAME=`echo $PIDFILE | cut -c14-`
     NAME=${NAME%%.pid}
     log_daemon_msg "  Stopping VPN '$NAME'"
@@ -280,9 +280,9 @@
       fi
       if test "x$AUTOVPN" = "x1" ; then
         # If it is autostarted, then it contributes to global status
-        status_of_proc -p /run/openvpn/${NAME}.pid openvpn "VPN '${NAME}'" || GLOBAL_STATUS=1
+        status_of_proc -p /run/wg-openvpn/${NAME}.pid wg-openvpn "VPN '${NAME}'" || GLOBAL_STATUS=1
       else
-        status_of_proc -p /run/openvpn/${NAME}.pid openvpn "VPN '${NAME}' (non autostarted)" || true
+        status_of_proc -p /run/wg-openvpn/${NAME}.pid wg-openvpn "VPN '${NAME}' (non autostarted)" || true
       fi
     done
   else
@@ -293,7 +293,7 @@
       NAME=$1
       if test -e $CONFIG_DIR/$NAME.conf ; then
         # Config exists
-        status_of_proc -p /run/openvpn/${NAME}.pid openvpn "VPN '${NAME}'" || GLOBAL_STATUS=1
+        status_of_proc -p /run/wg-openvpn/${NAME}.pid wg-openvpn "VPN '${NAME}'" || GLOBAL_STATUS=1
       else
         # Config does not exist
         log_warning_msg "VPN '$NAME': missing $CONFIG_DIR/$NAME.conf file !"
Index: openvpn-2.3.2/debian/openvpn.if-up.d
===================================================================
--- openvpn-2.3.2.orig/debian/openvpn.if-up.d	2014-02-04 23:29:42.000000000 +0900
+++ openvpn-2.3.2/debian/openvpn.if-up.d	2016-04-14 13:48:20.248126226 +0900
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-OPENVPN=/etc/init.d/openvpn
+OPENVPN=/etc/init.d/wg-openvpn
 
 if [ ! -x $OPENVPN ]; then
   exit 0
